app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/ConferenceTest.kt

```diff
@@ -24,36 +17,37 @@ class ConferenceTest {
     }
 
     @Test
-    fun `calculateTimeFrame with frab data spanning multiple days`() {
+    fun `ofSessions with frab data spanning multiple days`() {
         val opening = createSession("Opening", duration = 30, dateUtc = 1536332400000L) // 2018-09-07T17:00:00+02:00
         val closing = createSession("Closing", duration = 30, dateUtc = 1536504300000L) // 2018-09-09T16:45:00+02:00
-        val (firstSessionStartsAt, lastSessionEndsAt) = createConference(opening, closing)
-        assertThat(firstSessionStartsAt).isEqualTo(17 * 60 - 2 * 60) // 17:00h -2h zone offset = 15:00h
-        assertThat(lastSessionEndsAt).isEqualTo(17 * 60 + 15 - 2 * 60 + MINUTES_OF_ONE_DAY) // -> 17:15 -2h zone offset + day switch
+        val (firstSessionStartsAt, lastSessionEndsAt, spansMultipleDays) = createConference(opening, closing)
+        val firstSessionStartsAtMinutes = firstSessionStartsAt.minuteOfDay
+        val minutesToAdd = if (spansMultipleDays) MINUTES_OF_ONE_DAY else 0
+        val lastSessionEndsAtMinutes = lastSessionEndsAt.minuteOfDay + minutesToAdd
+        assertThat(firstSessionStartsAtMinutes).isEqualTo(17 * 60 - 2 * 60) // 17:00h -2h zone offset = 15:00h
+        assertThat(lastSessionEndsAtMinutes).isEqualTo(17 * 60 + 15 - 2 * 60 + MINUTES_OF_ONE_DAY) // -> 17:15 -2h zone offset + day switch
```

Comment:
I'd duplicate this test with the following dateUtc values:
1616857200000
1616940000000

first session starts at minutes should then be 15 * 60 (16:00 to UTC in winter), last session ends at Minutes should be 14*60+30 (16:30 to UTC in summer)

This is the hardest case I imagine, happend on easterhegg some years ago.

Reply:
Thanks. I added the scenario.

I also added daylight saving time scenarios to `ScrollAmountCalculatorTimeZoneOffsetTest` and `TimeTextViewParameterTest`. The latter seem to be wrong. Do you spot the error?

Reply:
:bulb: Clarification for myself: My confusion with regards to `TimeTextViewParameterTest` came from the fact that all calculation happens in UTC. Therefore, the time column segment do not represent the times according to winter nor summer time. What I still need to implement is that all rendering happens depending on the time zone offset of the device.

Reply:
Do you have an idea how I can ask the device what the time zone offset will be at the time when a future session takes place (using ThreeTenBp)? I can get the device time zone via `ZoneId.systemDefault()`.

Reply:
@a-tze I rebased and rewrote the branch based on my work in #370. It would be great if you take a look again.

Reply:
No idea how to do it with this actual library, but in general you would take a datetime with offset (or UTC) and ask for it to be converted in the systems timezone (e.g. Europe/Berlin), which results in another datetime with offset. Then you use the offset from that returned datetime (+1h oder +2h).

Reply:
This looks good to me! Just one comment to use october instead of (or additionally to) march to have even more fun with ambiguous time formats ;)

Reply:
My question from 04.02. is outdated. I forgot to write this here. Thank you anyway for your idea.